From 91dcbe36f2fbeadc96362c7ad28c18eca90c44e6 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Wed, 29 Mar 2023 10:52:43 +0200
Subject: [PATCH 48/N] CI: build with --disable-dependency-tracking

We only build once in CI, so dependency tracking isn't needed.
This saves 3-4 minutes in CI.
---
 .github/workflows/build.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.github/workflows/build.yaml b/.github/workflows/build.yaml
index 5069dbd..1775bb9 100644
--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -21,7 +21,7 @@ jobs:
         shell: msys2 {0}
         run: |
           (cd winsup && ./autogen.sh)
-          ./configure
+          ./configure --disable-dependency-tracking
           make -j8
 
       - name: Install
